client/shipping-label/views/purchase/steps/packages/index.js

```diff
@@ -7,15 +7,18 @@ import { sprintf } from 'sprintf-js';
 import StepContainer from '../../step-container';
 
 const PackagesStep = ( { values, storeOptions, labelActions, errors, expanded } ) => {
-	const isValid = 0 < values[ 0 ].weight;
+	const packageIds = Object.keys( values );
+	const firstPackageId = packageIds[ 0 ];
+	const firstPackage = values[ firstPackageId ];
+	const isValid = 0 < firstPackage.weight;
```

Comment:
You should check here if _any_ weight is invalid. I know, my bad, I implemented this supporting only 1 package, but since now it supports multiple, better to do it well.